content/en/serverless/configuration/_index.md

```diff
@@ -632,6 +632,14 @@ To migrate, compare the [installation instructions using the Datadog Lambda Exte
 
 **Note**: Datadog recommends migrating your dev and staging applications first and migrating production applications one by one.
 
+## Migrating between x86 to arm64 with the Datadog Lambda Extension
+
+The Datadog Extension is a compiled binary, and is available in both x86 and arm64 variants. If you are migrating an x86 Lambda function to arm64 (or arm64 to x86) using a deployment tool like CDK, Serverless Framework, or SAM; you should ensure that the you're service integration (API Gateway, SNS, Kinesis, etc) is configured to use a Lambda function versions or aliases, otherwise the function may be unavailable for a few seconds during deployment (approximately 10 seconds).
+
+This is due to the fact that migrating a Lambda function from x86 to arm64 consists of two parallel API calls, `updateFunction` and `updateFunctionConfiguration`. During these calls, there is a brief window where the Lambda `updateFunction` call has completed and the code is updated to use the new architecture while the `updateFunctionConfiguration` call has not yet completed, so the old architecture is still configured for the Extension.
```

Comment:
```suggestion
This happens because migrating a Lambda function from x86 to arm64 consists of two parallel API calls, `updateFunction` and `updateFunctionConfiguration`. During these calls, there is a brief window where the Lambda `updateFunction` call has completed and the code is updated to use the new architecture while the `updateFunctionConfiguration` call has not yet completed, so the old architecture is still configured for the Extension.
```